packages/utilities/src/index.ts

```diff
@@ -7,3 +7,5 @@
 
 export * from './dateTimeFormat';
 export * from './overflowHandler';
+export * from '@internationalized/number';
+export type * from '@internationalized/number';
```

Comment:
Intentionally avoiding a `peerDependency` by doing this, to avoid issues like we've had with `react-is`.